i2c/Cargo.toml

```diff
@@ -9,3 +9,11 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+clap = { version = "2.33.3",  features = ["yaml"] }
+epoll = "4.3"
```

Comment:
Can you use vmm-sys-util instead? We have an epoll wrapper there that is relatively safer than the `epoll` as it checks the results of unsafe functions.

Reply:
> Can you use vmm-sys-util instead? We have an epoll wrapper there that is relatively safer than the `epoll` as it checks the results of unsafe functions.

I used epoll instead as https://github.com/rust-vmm/vhost-user-backend uses the same and my repo depends on a function prototype from there.

Reply:
Is that fine @andreeaflorescu  ?

Reply:
Oh, I see. Okay.